backend/djangoindia/celery.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import, unicode_literals
+import os
+from celery import Celery
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'djangoindia.settings')
```

Comment:
This should solve the problem of task not getting discovered.
```suggestion
os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'djangoindia.settings.local')
```